modules/table/src/main/java/org/apache/ignite/internal/table/distributed/command/response/MultiRowsResponse.java

```diff
@@ -22,18 +22,11 @@
 import java.util.List;
 import org.apache.ignite.internal.schema.BinaryRow;
 import org.apache.ignite.internal.table.distributed.command.CommandUtils;
-import org.apache.ignite.internal.table.distributed.command.DeleteAllCommand;
-import org.apache.ignite.internal.table.distributed.command.DeleteExactAllCommand;
-import org.apache.ignite.internal.table.distributed.command.GetAllCommand;
-import org.apache.ignite.internal.table.distributed.command.InsertAllCommand;
 
 /**
  * This class represents a response object that contains a collection {@link BinaryRow} from a batch operation.
  *
- * @see GetAllCommand
- * @see DeleteAllCommand
- * @see InsertAllCommand
- * @see DeleteExactAllCommand
+ * @see org.apache.ignite.internal.table.distributed.command.UpdateAllCommand
```

Comment:
please use the short version of `UpdateAllCommand` with the corresponding import

Reply:
Fixed.